libgit2-sys/lib.rs

```diff
@@ -79,6 +79,16 @@ impl Clone for git_strarray {
 }
 
 #[repr(C)]
+#[derive(Copy)]
+pub struct git_oidarray {
+    pub ids: *mut git_oid,
+    pub count: size_t,
+}
+impl Clone for git_oidarray {
+    fn clone(&self) -> git_oidarray { *self }
+}
```

Comment:
Could this omit the `Clone + Copy` for now? Looking below it looks like it's not required at least?